sde-cookbook/dlt-viewer/install_dlt-viewer.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) Luxoft 2018
+#
+# All rights reserved
+#
+
+DLT_VIEWER_TMP_DIR=/tmp/github
+
+DLT_REPO_NAME="dlt-viewer"
+
+mkdir -p ${DLT_VIEWER_TMP_DIR}
+cd ${DLT_VIEWER_TMP_DIR}
+git clone https://github.com/GENIVI/${DLT_REPO_NAME}
```

Comment:
Maybe we should use a fixed revision in order to have reproducable builds?

Reply:
In general, reproducible builds are required in many projects and PELUX Baseline as a reference implementation should also follow best practices to support this.

Reply:
Ok.